emsdk.py

```diff
@@ -2775,7 +2770,14 @@ def extract_bool_arg(name):
 
   arg_old = extract_bool_arg('--old')
   arg_uses = extract_bool_arg('--uses')
+  arg_permanent = extract_bool_arg('--permanent')
   arg_global = extract_bool_arg('--global')
+  if arg_global:
+    print('--global is deprecated. Use `--system` to set the environment variables for all users')
```

Comment:
Why not print the warning but continue to do the right thing and set arg_system here?  We can phase it out eventually.

Reply:
Fixed in https://github.com/emscripten-core/emsdk/pull/621/commits/25efaf464a58404ecc72d39342a375df6a43f866

Reply:
Sorry I think this still doesn't work becaue arg_system is assigned on the line below.

Reply:
I think if you just move it below it should work.

Reply:
It is fixed now.